docs/content/docs/deep/summaryTelemetry.md

```diff
@@ -175,35 +163,43 @@ This should not even be possible, but it means that an unhandled error was raise
 > Event Prefix: `Summarizer:Running:`
 
 - `summarizeCount` - the number of summarize attempts this client has made. This can be used to correlate events for individual summary attempts.
+- `summarizerSuccessfulAttempts` - the number of successful summaries this summarizer instance has performed
```

Comment:
Here, it would be nice to mention relationships of properties. I assume (from just that section) that diff of these two counters is number of unsuccessful summary attempts, right?